pixl_core/src/core/uploader/_dicomweb.py

```diff
@@ -36,13 +36,15 @@ def __init__(self, project_slug: str, keyvault_alias: Optional[str]) -> None:
     def _set_config(self) -> None:
         # Use the Azure KV alias as prefix if it exists, otherwise use the project name
         az_prefix = self.keyvault_alias
-        az_prefix = az_prefix if az_prefix else self.project_slug
+        self.az_prefix = az_prefix if az_prefix else self.project_slug
 
-        self.user = self.keyvault.fetch_secret(f"{az_prefix}--dicomweb--username")
-        self.password = self.keyvault.fetch_secret(f"{az_prefix}--dicomweb--password")
+        self.orthanc_user = config("ORTHANC_USERNAME")
+        self.orthanc_password = config("ORTHANC_PASSWORD")
         self.orthanc_url = config("ORTHANC_URL")
```

Comment:
Slightly idle thought: how do we make sure we're not accidentally creating a route from `orthanc-raw` to DicomWeb?
When this code meets my change it will be potentially a different ORTHANC_URL var as we're now running in export api. Will need to be sure it's the right one.

Reply:
> how do we make sure we're not accidentally creating a route from orthanc-raw to DicomWeb?

Is that relevant? Since this code should only be run in `orthanc-anon`?

Reply:
Shouldn't this code be in export-api eventually?

Reply:
Yeah true, can you maybe add a note in your PR as a reminder so I can double check when merging with your work?

Reply:
Ah wait, doesn't this fall under the bit that you suggested should be moved to export-api. It was more awkward than I thought due to the ParquetExport class being called from both export and CLI.

Reply:
> > how do we make sure we're not accidentally creating a route from orthanc-raw to DicomWeb?
> 
> Is that relevant? Since this code should only be run in `orthanc-anon`?

Should, yes. But there's nothing obvious enforcing it and it would be quite bad if it did.

Reply:
Alright, will address this as part of the merge with #370